.rubocop.yml

```diff
@@ -123,3 +123,6 @@ Style/TrivialAccessors:
 
 Style/WordArray:
   Enabled: false
+
+Style/ColonMethodCall:
+  Enabled: false
```

Comment:
We also don't want to merge anything that breaks liquid-c or massively
increases it's complexity.

On Thursday, 24 March 2016, Florian Weingarten <notifications@github.com>
wrote:

> In .rubocop.yml
> <https://github.com/Shopify/liquid/pull/729#discussion_r57313433>:
>
> > @@ -123,3 +123,6 @@ Style/TrivialAccessors:
> >
> >  Style/WordArray:
> >    Enabled: false
> > +
> > +Style/ColonMethodCall:
> > +  Enabled: false
>
> why this? can you just use the regular dot notation please?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Shopify/liquid/pull/729/files/76e9aab5b8356975883dc0dd229abd9e276a88b0#r57313433>
>